### PR TITLE
new strings for new invitations

### PR DESCRIPTION
**Pull request by bbassett**

### CODE REVIEW

File: priv/auth-ui/en.json [hunk 1]
```diff
@@ -21,23 +21,29 @@
...
     "password_label": "Password",
     "password_placeholder": "8 characters minimum",
-    "signup_name": "SIGN UP",
     "signup_free": "First sign up for a free Boombox account.",
-    "submit_signin": "SIGN IN",
-    "submit_signup": "SIGN UP & CONTINUE",
+    "signup_name": "SIGN UP",
+    "signup_to_join_hub": "Signup to join the %{hubName} Hub",
+    "signup_to_join_org": "Signup to join the %{hubName} Organization",
+    "submit_signin": "LOG IN",
```
Comment (jakerichan) [inline priv/auth-ui/en.json:41]: if these are for buttons, we should use `submit_signin_label`

File: priv/auth-ui/en.json [hunk 2]
```diff
@@ -21,23 +21,29 @@
...
+    "submit_signup": "SIGN UP",
```
Comment (jakerichan) [inline priv/auth-ui/en.json:42]: and `submit_signup_label`

File: priv/auth-ui/en.json (hunk 1 above)
Comment (jakerichan) [inline priv/auth-ui/en.json:41]: Also, text transformation should happen in CSS. All these that are all capitals should just have the first letter capital

**Comment by jakerichan** [comment]
Actually, if you just moved them you don't have to worry about updating them

**Comment by jakerichan** [comment]
👍

**Comment by bbassett** [comment]
yeah, there are a LOT of translations that are capitalized in the languages repo, need to fix all of those. a lot of them were just moved, i'll see how much i can clean stuff up

**Comment by bbassett** [comment]
eh, everything that's 'wrong' was just moved for alphabetization, so i'm just going to merge it

**Pull request merged**